legend-engine-test-runner-service/src/test/resources/testable/legend-testable-service-M2M.pure

```diff
@@ -0,0 +1,137 @@
+###Data
+Data data::FirmData
+{
+  ExternalFormat
+  #{
+    contentType: 'application/json';
+    data: '{\n  "employees": [\n    {\n      "firstName": "John",\n      "lastName": "Smith"\n    }\n  ],\n  "legalName": "Apple",\n  "type": "llc"\n}';
+  }#
+}
+
+
+###Service
+Service mapping::ModelToModelMappingService
+{
+  pattern: '/09c0a540-04e6-4392-a979-fa90054530b5';
+  documentation: '';
+  autoActivateUpdates: true;
+  execution: Single
+  {
+    query: |model::target::_Firm.all()->graphFetchChecked(#{model::target::_Firm{name,type,employees{fullName}}}#)->serialize(#{model::target::_Firm{name,type,employees{fullName}}}#);
+    mapping: mapping::ModelToModelMapping;
+    runtime:
+    #{
+      mappings:
+      [
+        mapping::ModelToModelMapping
+      ];
+      connections:
+      [
+        ModelStore:
+        [
+          connection_1:
+          #{
+            JsonModelConnection
+            {
+              class: model::Firm;
+              url: 'data:application/json,%7B%22employees%22%3A%5B%7B%22firstName%22%3A%22firstName%2059%22%2C%22lastName%22%3A%22lastName%2050%22%7D%5D%2C%22legalName%22%3A%22legalName%2011%22%2C%22type%22%3A%22llc%22%7D';
+            }
+          }#
+        ]
+      ];
+    }#;
+  }
+  testSuites:
+  [
+    testSuite1:
```

Comment:
do we not already have this test?
Sorry I may have missed it

Reply:
yeah right. we do. removing it thanks.